resources/fixtures/Robe/Robe-Pointe.qxf

```diff
@@ -219,7 +238,7 @@
  </Channel>
  <Channel Name="Dimmer" Preset="IntensityDimmer"/>
  <Channel Name="Dimmer fine" Preset="IntensityDimmerFine"/>
- <Mode Name="Main">
+ <Mode Name="Mode 1">
```

Comment:
Having mode in the name is considered an error by the fixturevalidator so this and similar changes is necessary.

Reply:
@mcallegari @joshuaurieli @kluszon This comment is still not addressed, although the PR is now merged.

Reply:
@FloEdelmann too fast! See my comment. I'm gonna fix errors myself.